guides/hack/01-getting-started/03-starting-a-real-project.md

```diff
@@ -4,7 +4,7 @@ Real projects generally aren't a single file in isolation; they tend to have
 dependencies such as the [Hack Standard Library], and various optional tools.
 
 A good starting point is to:
-- [install Composer]
+- [install Composer](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-macos)
```

Comment:
This was/is a link reference; either update the reference definition at the bottom of the file and remove the inline URI here, or delete the reference definition at the bottom of the file